jina/enums.py

```diff
@@ -321,6 +321,12 @@ def is_keep(self) -> bool:
         """
         return self.value in {0, 2}
 
+class InfrastructureType(BetterEnum):
+    """Infrastructure where the Flow runs on."""
+
+    LOCAL = 0  # flow is started locally
```

Comment:
I would change the name, I would call Jina vs K8s. Local is not correct because Jina can be distributed

Reply:
Also in the comments do not refer to Flow but to Pods or services